src/PopUp.tsx

```diff
@@ -97,7 +97,7 @@ export const PopUp = ({
         </PopUpPlot>
         <PopUpPoster src={selected.Poster} alt=""></PopUpPoster>
       </PopUpWindow>
-      <PopUpBackButton onClick={() => backClick(false)}>Back</PopUpBackButton>
+      <PopUpBackButton onClick={() => backClick()}>Back</PopUpBackButton>
```

Comment:
Is this purely sending back up that the back button is clicked?

Reply:
yeah exactly, the fact that you had it hardcoded as passing `false` indicated to me that you didn't really need the argument at all